oak_restricted_kernel/src/lib.rs

```diff
@@ -89,11 +89,21 @@ pub fn start_kernel(info: &BootParams) -> Box<dyn Channel> {
     // Physical frame allocator: support up to 128 GiB of memory, for now.
     let mut frame_allocator = mm::init::<1024>(info.e820_table(), program_headers);
 
-    let mapper = mm::init_paging(&mut frame_allocator, program_headers).unwrap();
+    let mut mapper = mm::init_paging(&mut frame_allocator, program_headers).unwrap();
+
+    // Allocate a section for guest-host communication (without the `ENCRYPTED` bit set)
+    let guest_host_frame: PhysFrame<Size2MiB> = frame_allocator.allocate_frame().unwrap();
```

Comment:
For now 2MiB is fine, but we might need a bit more in future if we want to instantiate the virtio vsock in this memory as well, as it uses just over 2MiB for it buffers.